lampo-common/src/keys.rs

```diff
@@ -3,40 +3,49 @@ use std::{sync::Arc, time::SystemTime};
 use bitcoin::secp256k1::{Secp256k1, SecretKey};
 use lightning::sign::{InMemorySigner, NodeSigner, OutputSpender, SignerProvider};
 
-use crate::ldk::sign::{EntropySource, KeysManager};
+use crate::{conf::LampoConf, ldk::sign::{EntropySource, KeysManager}};
```

Comment:
There are wrong imports, it is from the day 1 of the internship that I am trying to tell you this.
```suggestion

use crate::conf::LampoConf; 
use crate::ldk::sign::{EntropySource, KeysManager};
```